RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -0,0 +1,192 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ManageSubscriptionsView.swift
+//
+//
+//  Created by Andrés Boedo on 5/3/24.
+//
+
+import RevenueCat
+import SwiftUI
+
+#if !os(macOS) && !os(tvOS) && !os(watchOS) && !os(visionOS)
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+@available(visionOS, unavailable)
+struct ManageSubscriptionsView: View {
+
+    @Environment(\.openURL)
+    var openURL
+
+    @StateObject
+    private var viewModel = ManageSubscriptionsViewModel()
+
+    init() { }
+
+    fileprivate init(viewModel: ManageSubscriptionsViewModel) {
+        self._viewModel = .init(wrappedValue: viewModel)
+    }
+
+    var body: some View {
+        VStack {
+            if viewModel.isLoaded {
+                HeaderView(viewModel: viewModel)
+
+                if let subscriptionInformation = self.viewModel.subscriptionInformation {
+                    SubscriptionDetailsView(subscriptionInformation: subscriptionInformation,
+                                            refundRequestStatusMessage: viewModel.refundRequestStatusMessage)
+                }
+
+                Spacer()
+
+                ManageSubscriptionsButtonsView(viewModel: viewModel)
+            } else {
+                ProgressView()
+                    .progressViewStyle(CircularProgressViewStyle())
+            }
+        }
+        .task {
+            await checkAndLoadInformation()
+        }
+    }
+
+}
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+@available(visionOS, unavailable)
+private extension ManageSubscriptionsView {
+
+    func checkAndLoadInformation() async {
```

Comment:
Maybe clearer name based on [Apple's similar situation](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621446-loadviewifneeded).
```suggestion
    func loadInformationIfNeeded() async {
```